util/coverage/coverage.js

```diff
@@ -47,41 +47,59 @@ const exVersionToShortName = {
   '0': {
     en: 'ARR',
     de: 'ARR',
+    ja: '新生',
+    cn: '2.X',
   },
   '1': {
     en: 'HW',
     de: 'HW',
+    ja: '蒼天',
+    cn: '3.X',
   },
   '2': {
     en: 'SB',
     de: 'SB',
+    ja: '紅蓮',
+    cn: '4.X',
   },
   '3': {
     en: 'ShB',
     de: 'ShB',
+    ja: '漆黒',
+    cn: '5.X',
   },
 };
 
 const contentTypeToLabel = {
   [ContentType.Raids]: {
     en: 'Raid',
     de: 'Raid',
+    ja: 'レイド',
+    cn: 'Raid',
   },
   [ContentType.Trials]: {
     en: 'Trial',
     de: 'Prfng',
+    ja: '討伐戦',
+    cn: '蛮神',
   },
   [ContentType.UltimateRaids]: {
     en: 'Ult',
     de: 'Ult',
+    ja: '絶',
+    cn: '绝',
   },
   [ContentType.Dungeons]: {
     en: 'Dgn',
     de: 'Dgn',
+    ja: 'ダンジョン',
```

Comment:
@r4ha Is there any abbreviation of Dungeon in Japanese? This should be shorten as possible.

Reply:
ID (Instanced Dungeons) is good here I guess.

Reply:
Oh, I even forgot this!
Thank you!